### DISCUSSION TITLE
TypeError

### DISCUSSION THREAD

**Discussion by marcelosantos32** [community/community#123852] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm getting the error 'TypeError: undefined is not a function' when trying to call a method on an object. What could be causing this?

**Comment by gabrielbaltar** [comment]
Check object existence: Make sure the object you're trying to call the method on actually exists and has been properly initialized before trying to access its methods. Use console.log or a debugger to inspect the object.
Verify method existence: Confirm that the method you're trying to call actually exists on the object. Check the spelling and ensure that it was defined correctly in the class or object.

**Comment by ebndev** [comment]
Hi @marcelosantos32 & @gabrielbaltar, thanks for being a part of the GitHub Community.

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
